sopel/config/types.py

```diff
@@ -216,31 +219,43 @@ def __get__(self, instance, owner=None):
             # (and is really just more intuitive) to return the descriptor
             # instance here.
             return self
-
+        value = None
```

Comment:
Let's leave the blank line above this.

```suggestion

        value = None
```